README.md

```diff
@@ -1,16 +1,162 @@
 # deroll
 
 Deroll is a TypeScript framework for developing [Cartesi](https://cartesi.io) applications.
-The code below is a minimal application which just loops forever fetching inputs, but with no input handlers. In that case any input is `rejected`.
+The code below is a minimal application which just loops forever fetching inputs, but with no input handlers.
+In that case any input is `rejected`.
 
-```typescript
+Input handles are functions that take an input and return a promise of an output.
+
+## Quick Start
+
+### Prerequisites
+
+Make sure you have Node.js and Yarn installed. You can download them from [nodejs.org](https://nodejs.org/) and [yarnpkg.com](https://yarnpkg.com/).
+
+### Install sunodo
```

Comment:
Instead of replicating the content I think I prefer to link the original documentation.

Reply:
plz, verify if I have understood correctly...